### PR TITLE
Delete boilerplate, Add calculator

### PR DESCRIPTION
**Pull request by AlyAtalla**
Deleted all the boilerplate from Create React App
Created a new directory called components.
Inside components, created a new Calculator.js file.
In Calculator.js, created a React component that matches the design
Implemented the design only (JSX and CSS)
Developed the UI of the calculator

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <div className="button-row">
+          <button type="button" className="button" onClick={() => handleClick(1)}>1</button>
+          <button type="button" className="button" onClick={() => handleClick(2)}>2</button>
+          <button type="button" className="button" onClick={() => handleClick(3)}>3</button>
+          <button type="button" className="button operator-button">+</button>
+        </div>
+        <div className="button-row">
+          <button type="button" className="button" onClick={() => handleClick(0)}>0</button>
+          <button type="button" className="button" onClick={() => handleClick('.')}>.</button>
+          <button type="button" className="button operator-button">=</button>
```
Comment (iambenkis) [inline src/components/Calculator.js]: - [ ] Kindly make sure that the design match the given design template, as it is required in the style guideline 👍 

Your design 

Capture

Design to match

Capture1

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @AlyAtalla 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  Has Calculator component
✔️ Good design  

## Suggested changes

- [ ] It could be better if you fix the broken link attached to the MIT license, remember that broken links make your documentation unprofessional.
 
Capture


_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/Calculator.js (hunk 1 above)
Comment (AlyAtalla) [reply src/components/Calculator.js]: thank you

File: src/components/Calculator.css [hunk 2]
```diff
@@ -46,3 +46,7 @@ button {
 button:hover {
   background-color: #ddd;
 }
+
+.button.zero-button {
+width: 9rem;
+}
```
Comment (ridwanediallo) [inline src/components/Calculator.css]: - [x] Excellent job implementing the previous reviewer's suggestions 👏. However, after making the changes, the equal button has shifted slightly to the right. So make sure it's properly aligned with those on top, and I'd highly recommend updating the **width** to **8.75rem** to give your UI a nicer look.  :ok_hand:  😊 

```
.button.zero-button {
   width: 8.75rem;
}
```

#### Your calculator currently :point_down: 

Screenshot from 2023-08-07 15-57-00


#### Your calculator when you update `button.zero-button` 's width to `8.75rem` :point_down: 
Screenshot from 2023-08-07 16-28-01

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @AlyAtalla  


AlmostThereReachGIF

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- Lincense link is correctly woking. ✔


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.css (hunk 2 above)
Comment (AlyAtalla) [reply src/components/Calculator.css]: thank you

**Review by Owusu-Desmond — APPROVED** [review]
Hi @AlyAtalla , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**